app/src/main/java/com/cornellappdev/uplift/ui/screens/GymDetailScreen.kt

```diff
@@ -36,25 +38,32 @@ import java.util.*
  */
 @Composable
 fun GymDetailScreen(
-    gymDetailViewModel: GymDetailViewModel
+    gymDetailViewModel: GymDetailViewModel = viewModel()
 ) {
     val gym by gymDetailViewModel.gymFlow.collectAsState()
     val day = ((Calendar.getInstance().get(Calendar.DAY_OF_WEEK) - 2) + 7) % 7
 
+    val scrollState = rememberScrollState()
+
     Column(
         modifier = Modifier
             .fillMaxSize()
-            .verticalScroll(rememberScrollState())
+            .verticalScroll(scrollState)
     ) {
         // Top Part
-        Box(modifier = Modifier.fillMaxWidth()) {
+        Box(modifier = Modifier
+            .fillMaxWidth()
+            .graphicsLayer {
+                alpha = 1 - (scrollState.value.toFloat() / 2000)
```

Comment:
Is there a better way to do this other than dividing by 2000? Like dividing by the length of the screen

Reply:
Changed to using screen height in commit [d466ef2](https://github.com/cuappdev/uplift-android/pull/3/commits/d466ef23c45556317befcd4552bd65c4f2839eac).